spec/lib/openregister_spec.rb

```diff
@@ -607,6 +610,15 @@ def stub_tsv_request url, fixture, headers: {}
           expect(display[1]).to eq "GARSTON ENTERPRISE ACADEMY 2009-02-09 - 2009-12-21"
         end
       end
+
+      describe 'retrieve a register\'s entries first page only via #_entries' do
+        it 'returns entries as Ruby objects' do
+          entries = OpenRegister.registers[1]._entries
+          expect(entries).to be_an(Array)
```

Comment:
Asserting on the type seems redundant given that what you actually care about is that it returns something that implements `Enumerable` which is implicitly tested in the next line.